src/Tizen.NUI/src/public/RenderEffects/RenderEffect.cs

```diff
@@ -51,5 +63,38 @@ public static RenderEffect CreateBackgroundBlurEffect(float blurRadius, bool blu
         {
             return new RenderEffect(Interop.BackgroundBlurEffect.New((uint)Math.Round(blurRadius, 0), blurOnce));
         }
+
+        /// <summary>
+        /// Create a mask effect
+        /// </summary>
+        /// <remarks>
+        /// Created RenderEffect is immutable.
+        /// </remarks>
+        /// <param name="control">The mask source control.</param>
+        /// <returns>mask effect with given control.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
We are now in refactoring of RenderEffect to make it mutable object.
Please change this after the refactoring